yarn-project/sequencer-client/src/config.ts

```diff
@@ -30,6 +29,12 @@ export function getConfigEnvVars(): SequencerClientConfig {
     CONTRACT_DEPLOYMENT_EMITTER_ADDRESS,
   } = process.env;
 
+  const publisherPrivateKey: `0x${string}` = `0x${
+    SEQ_PUBLISHER_PRIVATE_KEY
+      ? SEQ_PUBLISHER_PRIVATE_KEY.replace('0x', '')
+      : '0000000000000000000000000000000000000000000000000000000000000000'
+  }`;
```

Comment:
Is this because we need to pad to 32 bytes? If so, let's create a `padTo32ByteString` function if we don't have one already, rather than doing this blindly. Otherwise, if the user is already passing in a 32-byte value as an env var, we'll be adding an extra 12 bytes without noticing.

Reply:
It's not because of padding. What this does is simply check if the private key is defined and if not set "0x 0000000000000000000000000000000000000000000000000000000000000000" as the private key value (this is what we do on [master](https://github.com/AztecProtocol/aztec-packages/blob/46378068647a47232922f93b2ac5d5f5370fd698/yarn-project/sequencer-client/src/config.ts#L48) as well). I changed the ethereum private key type to `0x${string}` because there is no longer a more concrete type compatible with it.